docs/manual.xml.head

```diff
@@ -12073,8 +12073,25 @@ set imap_oauth_refresh_command="/path/to/oauth2.py --quiet --user=[email_address
 </screen>
 
       <para>
-        Substitute pop or smtp for imap in the above example to configure for
-        those.
+        For Office365, you can use the <literal>mutt_oauth2.py</literal> script
+        written by Alexander Perlis:
+        <ulink url="https://gitlab.com/muttmua/mutt/-/blob/master/contrib/mutt_oauth2.py">https://gitlab.com/muttmua/mutt/-/blob/master/contrib/mutt_oauth2.py</ulink>
```

Comment:
Do we perhaps want to add these two files to our contrib directory?

Reply:
Imported them!